docs/api/webview-tag.md

```diff
@@ -823,6 +823,28 @@ this purpose.
 
 Calling `event.preventDefault()` does __NOT__ have any effect.
 
+### Event: 'will-frame-navigate'
+
+Returns:
+
+* `url` string
+* `isMainFrame` boolean
+* `frameProcessId` Integer
+* `frameRoutingId` Integer
```

Comment:
this should be `frame`

Reply:
This is the event forwarded to the renderer's webview wherein WebFrameMain doesn't exist.

Link to earlier discussion: https://github.com/electron/electron/pull/34418#discussion_r890447734

Reply:
ah sure... hmm okay. What would the renderer actually do with this information?

Reply:
Including `frameProcessId` and `frameRoutingId` would be consistent with other events e.g. `will-redirect`, `did-start-navigation`, `did-redirect-navigation`. So existing code that has these handles would likely not have to do much refactoring to use this new event as well

Reply:
Renderer code could pass these to Main code which could call [webFrameMain.fromId](https://www.electronjs.org/docs/latest/api/web-frame-main#webframemainfromidprocessid-routingid). You could probably accomplish something similar by getting the `<webview>`'s `webContents` from the main process and listening to `will-frame-navigate` directly, but I think that might be less ergonomic if Renderer logic is needed while handling the `will-frame-navigate` event.

That said, I don't have a specific concrete use-case for this event on `<webview>`. I just added it for completeness' sake. If we want to wait to add `<webview>.will-frame-navigate` until a more concrete use-case can be articulated, I would be OK with that.

Reply:
yeah, seems like reasonable practice to add. if it's consistent with other APIs on the webview tag i'm ok with this.